setup.py

```diff
@@ -428,7 +437,7 @@ def setup_package():
                 [
                     "-DNMODL_ENABLE_PYTHON_BINDINGS=ON",
                 ]
-                if Components.CORENRN
+                if (Components.CORENRN or Components.NMODL_DOCS)
```

Comment:
I am not 100% sure if we need all of this stuff with `NMODL_DOCS` now as there should just be one way of building the docs, and that's via CMake.